docs/design/OUTPUT_DESIGN_TRANSACTIONS.md

```
@@ -0,0 +1,85 @@
+
+---
+
+## 概要
+
+CSVから取り込んだ明細データを一覧表示・カテゴリ管理・分析するモード。
+
+---
+
+## 画面構成
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)